aws/table_aws_ssm_document.go

```diff
@@ -387,6 +416,18 @@ func getAwsSSMDocumentAkas(ctx context.Context, d *plugin.QueryData, h *plugin.H
 	return []string{aka}, nil
 }
 
+func getgetAwsSSMDocumentArn(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	akas, err := getAwsSSMDocumentAkas(ctx, d, h)
```

Comment:
In this function, can we instead compose the ARN from the information we have available? This helps protect us against future changes in `akas` where the first entry isn't always the ARN